### ISSUE TITLE
How do I remove double slash in URLs?

### ISSUE THREAD

**Issue by danherd** [cloudcreativity/laravel-json-api#142]
With this configuration:

```
    'url' => [
        'host' => null,
        'namespace' => '/api',
        'name' => 'api:',
    ],
```

URLs look like this, which is fine:

```
        "links": {
            "self": "https://api.loc8um.dev/api/locums/DA49890F-1CB5-46F8-A501-57E3B22C0BFA"
        }
```

However, I don't want/need the '/api' bit, so I change the configuration to this:

```
    'url' => [
        'host' => null,
        'namespace' => '',
        'name' => 'api:',
    ],
```

And the URLs look like this:

```
        "links": {
            "self": "https://api.loc8um.dev//locums/94A14168-4B12-403C-B403-E852C4E8B6BE"
        }
```

And it does the same thing if I set the namespace to '/', null or false too.

So, how do I stop it double slashing?

**Comment by lindyhopchris** [comment]
Confirmed this is a bug... looking at fixing it now.

**Comment by lindyhopchris** [comment]
Fixed in `v0.11.5`
